wowup-electron/main.ts

```diff
@@ -230,7 +230,9 @@ function createWindow(): BrowserWindow {
   win.webContents.userAgent = USER_AGENT;
 
   win.once("ready-to-show", () => {
-    win.show();
+    var startMinimized = (process.argv || []).indexOf('--hidden') !== -1;
```

Comment:
Can we use something like minimist or yargs for argument parsing instead of hand making this?

Reply:
Ok, will do.

P.S.
Why do you hate manual args parsing? :D

Reply:
using a lib to do it will make it easier to add new args as we need and not have to overthink how to add them.